internal/controller/controllers/controller_event_wrapper.go

```diff
@@ -26,18 +26,18 @@ func NewControllerEventsWrapper(crdEventsHandler CRDEventsHandler, events events
 		events: events, db: db, log: log}
 }
 
-func (c *controllerEventsWrapper) AddEvent(ctx context.Context, clusterID strfmt.UUID, hostID *strfmt.UUID, severity string, msg string, eventTime time.Time, props ...interface{}) {
-	c.events.AddEvent(ctx, clusterID, hostID, severity, msg, eventTime, props)
+func (c *controllerEventsWrapper) V2AddEvent(ctx context.Context, clusterID *strfmt.UUID, hostID *strfmt.UUID, infraEnvID *strfmt.UUID, severity string, msg string, eventTime time.Time, props ...interface{}) {
+	c.events.V2AddEvent(ctx, clusterID, hostID, infraEnvID, severity, msg, eventTime, props)
 
 	if hostID != nil {
-		c.NotifyKubeApiHostEvent(clusterID, *hostID)
+		c.NotifyKubeApiHostEvent(common.StrFmtUUIDVal(infraEnvID), common.StrFmtUUIDVal(hostID))
 	} else {
-		c.NotifyKubeApiClusterEvent(clusterID)
+		c.NotifyKubeApiClusterEvent(common.StrFmtUUIDVal(clusterID))
 	}
 }
 
-func (c *controllerEventsWrapper) V2AddEvent(ctx context.Context, clusterID *strfmt.UUID, hostID *strfmt.UUID, infraEnvID *strfmt.UUID, severity string, msg string, eventTime time.Time, props ...interface{}) {
-	//TODO: Implement this instead of v1 AddEvent() when it get removed.
+func (c *controllerEventsWrapper) AddEvent(ctx context.Context, clusterID strfmt.UUID, hostID *strfmt.UUID, severity string, msg string, eventTime time.Time, props ...interface{}) {
+	//TODO: Remove this when V1 Events endpoint gets removed.
```

Comment:
Shouldn't we leave this implemented and remove it as part of the PR removing the v1 event handling? I'm failing to see the point of keeping this function definition if we are removing the whole implementation. Is there anything else calling it? 

Another option could be to have this `AddEvent` call `V2AddEvent` under-the-hood buit, at that point, we could also just keep `AddEvent` and have it use `V2`. no?

Reply:
I kept the function definition in place, just because it needs to comply with the interface.
It is not used by any other place in the kubeAPI controllers code.